src/go/configmanager/config_manager_test.go

```diff
@@ -1803,6 +1801,7 @@ func runTest(t *testing.T, opts options.ConfigGeneratorOptions, f func(*testEnv)
 
 	metadataFetcher := metadata.NewMockMetadataFetcher(mockMetadataServer.URL, time.Now())
 
+	opts.RootCertsPath = platform.GetFilePath(platform.RootCaCerts)
```

Comment:
this is unit test, I don't think it really reads the file

Reply:
It does read the file, which is not hermetic.

https://github.com/GoogleCloudPlatform/esp-v2/blob/master/src/go/configmanager/config_manager.go#L312

Reply:
-> platform.TestRootCaCerts

Reply:
Done